package.json

```diff
@@ -120,7 +120,8 @@
     "numeric": "^1.2.6",
     "plottable": "^3.9.0",
     "requirejs": "^2.3.6",
-    "rxjs": "^6.5.5",
+    "rxjs": "^7.0.0-beta.0",
+    "static-eval": "^2.1.0",
```

Comment:
we don't have direct usage of static-eval, right? What is this achieving? If anything, it should be devDependency.

Reply:
Correct, removed for now.

static-eval, bl, are transitive deps of weblas, which doesn't have any newer version.  Might be worth migrating off weblas someday, as it's not actively updated and we only use it for GPU-matmul.